ansible/vars/topo_appliance.yml

```diff
@@ -28,7 +28,7 @@ configuration:
       asn: 65200
       peers:
         65100:
-        - 10.0.0.32
```

Comment:
Do we want to make these changes permanent? If we can get the tests passing with the existing IPs, we should try and keep these the same to maintain consistency between other testbeds.